src/Provider/SessionServiceProvider.php

```diff
@@ -341,9 +341,9 @@ protected function parseConnections($options, $defaultHost, $defaultPort)
             }
         } elseif (isset($options['save_path'])) {
             foreach (explode(',', $options['save_path']) as $conn) {
-                $conn = new ParameterBag($conn);
-                $conn->set('uri', new Uri($conn));
-                $toParse[] = $conn;
+                $connBag = new ParameterBag();
+                $connBag->set('uri', new Uri($conn));
+                $toParse[] = $connBag;
             }
```

Comment:
@CarsonF an explanation: `$conn` is a string, but the constructor for Parameter bag expects an array. Also, you were passing the parameter bag into the `Uri` constructor and then setting that to 'uri' in the same ParameterBag. It didn't make sense to me (and threw an exception), but maybe I'm missing something.